integration-tests/soak/tests/ocr_test.go

```diff
@@ -0,0 +1,263 @@
+package soak_test
```

Comment:
there seems to be a lot of duplicated code from `smoke/ocr2_test.go`. is there any way to reduce the code duplication?

Reply:
Wrapped gauntlet contract deployments as well as env deployments to reduce duplication

Reply:
there's still quite a few duplicate parts:
- env var look ups
- job spec creation and node setup
- the entire section for inspecting the operating feed

Reply:
Decoupled except for "the entire section for inspecting the operating feed" as discussed will be done afterwards